tests/serial/time/explicit/parallel-coupling/ReadWriteScalarDataWithWaveformSampling.cpp

```diff
@@ -0,0 +1,131 @@
+#ifndef PRECICE_NO_MPI
+
+#include "testing/Testing.hpp"
+
+#include <precice/SolverInterface.hpp>
+#include <vector>
+
+using namespace precice;
+
+BOOST_AUTO_TEST_SUITE(PreciceTests)
+BOOST_AUTO_TEST_SUITE(Serial)
+BOOST_AUTO_TEST_SUITE(Time)
+BOOST_AUTO_TEST_SUITE(Explicit)
+BOOST_AUTO_TEST_SUITE(ParallelCoupling)
+
+/**
+ * @brief Test to run a simple coupling with sampling from the waveform.
+ */
+BOOST_AUTO_TEST_CASE(ReadWriteScalarDataWithWaveformSampling)
```

Comment:
You probably want to put the order zero into the name?

Reply:
Yes. This is fixed now.